base/_extends.scss

```diff
@@ -1,7 +1,16 @@
 //FONT SIZES
-$fonts : 10, 12, 14, 16, 16, 18, 20, 30, 40;
-@each $font in $fonts {
-  .font-#{$font}{
-    font-size: $font + 'px';
-  }
-}
\ No newline at end of file
+@for $i from 8 through 50 {
```

Comment:
can we make these values dynamic so that dev can enter values in variables only and it start working. I don't want devs to get deep into code.